.travis.yml

```diff
@@ -1,18 +1,11 @@
 language: ruby
+
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.0
-  - jruby
+  - 2.2.4
+  - 2.3.0
 
 gemfile:
-  - test/gemfiles/Gemfile.rails-3.2.x
```

Comment:
Yes, because it didn't support ruby 2.3. 
It should support 2.2 but it fails on minitest/autorun